editor/plugins/visual_shader_editor_plugin.cpp

```diff
@@ -1552,6 +1550,7 @@ void VisualShaderEditor::edit_shader(const Ref<Shader> &p_shader) {
 			_update_options_menu();
 			_update_preview();
 			_update_graph();
+			call_deferred("_restore_editor_state");
```

Comment:
```suggestion
			callable_mp(this, &VisualShaderEditor::_restore_editor_state).call_deferred();
```
And remove the bind.